.github/workflows/docker.yml

```
@@ -6,37 +6,48 @@ on:
     branches:
     - develop
 
+
+permissions:
+  contents: read
```

```suggestion
permissions: {}
```

Let's start with the minimum level of permissions outside all jobs, and be as explicit as we can be.